content/en/docs/tasks/administer-cluster/configure-multiple-schedulers.md

```diff
@@ -123,6 +123,14 @@ $ kubectl edit clusterrole system:kube-scheduler
     - update
 ```
 
+my-scheduler runs as the “default” service account in the “kube-system” namespace. To allow it run with super-user access, grant cluster-admin permissions to the “default” service account in the “kube-system” namespace.
```

Comment:
s/"default"/`default`
s/"kube-system"/`kube-system`

in other words, replace quotation marks with backticks so that these strings are formatted as code. We use quotation marks in the docs only for strings where strings must be programmatically denoted as such. (There are many violations, but we're trying to clean up ;) )

Reply:
done